.env.example

```
@@ -22,7 +22,7 @@ JWT_REFRESH_TTL_DAYS=14
 AUTH_ACCESS_COOKIE=access_token
 AUTH_REFRESH_COOKIE=refresh_token
 AUTH_COOKIE_DOMAIN=null
-AUTH_COOKIE_SECURE=true
+AUTH_COOKIE_SECURE=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_COOKIE_SECURE key should go before the AUTH_REFRESH_COOKIE key

(UnorderedKey)